service/project.go

```diff
@@ -89,6 +89,23 @@ func (uis *UIServer) projectPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Replace ChildProject IDs of PatchTriggerAliases with the ChildProject's Identifier
+	for i, t := range projRef.PatchTriggerAliases {
+		var childProject *model.ProjectRef
+		childProject, err = model.FindOneProjectRef(t.ChildProject)
+		if err != nil {
+			uis.LoggedError(w, r, http.StatusInternalServerError, err)
```

Comment:
You'll want to return after each of these errors, otherwise 106 will panic

Reply:
Thanks for catching this!